contracts/src/RIP7755Structs.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+/// @notice Low-level call specs representing the desired transaction on destination chain
```

Comment:
do we have good cause to put these on their own? https://github.com/coinbase/solidity-style-guide?tab=readme-ov-file#a-prefer-declaring-structs-and-errors-within-the-interface-contract-or-library-where-they-are-used

Reply:
All three structs will be used by the source chain contract as well so figured they could live in their own file. If you disagree I guess it would make sense to put `Call` and `CrossChainCall` in the source chain contract and `FulfillmentInfo` in the verifier contract. What do you think?

Reply:
I think think that reasonable CrossChainCall is universal and meets this criteria? 
``` If a struct or error is used across many files, with no interface, contract, or library reasonably being the "owner," then define them in their own file. Multiple structs and errors can be defined together in one file.```